pkg/proxy/iptables/proxier.go

```diff
@@ -763,32 +759,14 @@ func (proxier *Proxier) syncProxyRules() {
 			writeLine(proxier.filterChains, utiliptables.MakeChainLine(chainName))
 		}
 	}
-	for _, chainName := range []utiliptables.Chain{kubeServicesChain, kubeNodePortsChain, kubePostroutingChain, KubeMarkMasqChain} {
+	for _, chainName := range []utiliptables.Chain{kubeServicesChain, kubeNodePortsChain} {
 		if chain, ok := existingNATChains[chainName]; ok {
 			writeBytesLine(proxier.natChains, chain)
 		} else {
 			writeLine(proxier.natChains, utiliptables.MakeChainLine(chainName))
 		}
 	}
 
-	// Install the kubernetes-specific postrouting rules. We use a whole chain for
-	// this so that it is easier to flush and change, for example if the mark
-	// value should ever change.
-	writeLine(proxier.natRules, []string{
```

Comment:
We still need this as far as I can tell, where is this rule getting replaced in the iptables proxy?

Reply:
(We still need to use `KUBE-MARK-MASQ` in the proxy, we just shouldn't try to flush/delete the chain as part of `--cleanup`)

Reply:
you can see [kubelet_network_linux.go#L99-L104](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_network_linux.go#L99-L104) and [kubelet_network_linux.go#L90-L93](
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_network_linux.go#L90-L93), kubelet will create rules, it just like `KUBE-MARK-DROP`, iptables/ipvs also need `KUBE-MARK-DROP` rule, but it only create by kubelet at [kubelet_network_linux.go#L56-L59](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_network_linux.go#L56-L59), iptables/ipvs don't manage `KUBE-MARK-DROP`.

Reply:
not only clean up, we also shouldn't create `KUBE-MARK-MASQ` `KUBE-POSTROUTING` chains and rules in iptables/ipvs proxier.

Reply:
Can we open a separate issue for this and address in a separate PR please? Ideally this PR only fixes issues reported in #68447.

Reply:
ok